_i18n/es/home.html

```diff
@@ -0,0 +1,41 @@
+<div class="rich-text home-page">
```

Comment:
Also this entire page can be deleted, the `index.md` purely uses the translation keys now